### PR TITLE
docs: update upgrade to consul-dataplane docs on k8s

### PR DESCRIPTION
**Pull request by ishustava**
### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

**Review by david-yu — APPROVED** [review]

**Comment by david-yu** [comment]
Looks good thanks for the PR.

File: website/content/docs/k8s/upgrade/index.mdx [hunk 1]
```diff
@@ -228,6 +228,18 @@ If you upgrade Consul from a version that uses client agents to a version the us
...
+We need to do it to make sure that the ACL tokens that existing services are using are still valid while we are migrating them to `consul-dataplane`.
+Note that this token will need to be cleaned up manually.
+
+    ```bash
+    kubectl config set-context --current --namespace=<consul installation namespace>
+    INJECTOR_DEPLOYMENT=$(kg deploy -l "component=connect-injector" -o=jsonpath='{.items[0].metadata.name}')
+    kubectl patch deploy $INJECTOR_DEPLOYMENT --type='json' -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/lifecycle"}]'
+    ```
+
+This will trigger the deployment rollout. We need to wait for the rollout to complete before proceeding to next step.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/upgrade/index.mdx:241]:
```suggestion
You must that the ACL tokens used by existing services are are still valid during the migration to `consul-dataplane`. Note that you must remove the token manually after completing the migration.
The following command triggers the deployment rollout. Wait for the rollout to complete before proceeding to next step.

    ```bash
    kubectl config set-context --current --namespace=<consul installation namespace>
    INJECTOR_DEPLOYMENT=$(kg deploy -l "component=connect-injector" -o=jsonpath='{.items[0].metadata.name}')
    kubectl patch deploy $INJECTOR_DEPLOYMENT --type='json' -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/lifecycle"}]'
    ```
```
I assume that "we" refers to the practitioner, not HashiCorp, correct? And by clean up, I think we mean removal after the migration process, yes?

**Review by trujillo-adam — COMMENTED** [review]
A few suggestions for consistency

File: website/content/docs/k8s/upgrade/index.mdx (hunk 1 above)
Comment (ishustava) [reply website/content/docs/k8s/upgrade/index.mdx:241]: Yes for both. Thanks for the suggestion.

**Review by NicoletaPopoviciu — APPROVED** [review]

File: website/content/docs/k8s/upgrade/index.mdx [hunk 2]
```diff
@@ -228,6 +228,18 @@ If you upgrade Consul from a version that uses client agents to a version the us
...
+to make sure that the ACL tokens used by existing services are still valid during the migration to `consul-dataplane`.
+Note that you must remove the token manually after completing the migration.
+
+The following command triggers the deployment rollout. Wait for the rollout to complete before proceeding to next step.
+
+    ```bash
+    kubectl config set-context --current --namespace=<consul installation namespace>
+    INJECTOR_DEPLOYMENT=$(kg deploy -l "component=connect-injector" -o=jsonpath='{.items[0].metadata.name}')
+    kubectl patch deploy $INJECTOR_DEPLOYMENT --type='json' -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/lifecycle"}]'
+    ```
```
Comment (trujillo-adam) [inline website/content/docs/k8s/upgrade/index.mdx]:
```suggestion
    The following command triggers the deployment rollout. Wait for the rollout to complete before proceeding to next step.

    ```bash
    kubectl config set-context --current --namespace=<consul installation namespace>
    INJECTOR_DEPLOYMENT=$(kg deploy -l "component=connect-injector" -o=jsonpath='{.items[0].metadata.name}')
    kubectl patch deploy $INJECTOR_DEPLOYMENT --type='json' -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/lifecycle"}]'
    ```
```
Sorry, I think I messed up your formatting with my last suggestion

**Review by trujillo-adam — APPROVED** [review]
Sorry, I introduced a formatting issue in my last comment.

**Pull request merged**